### PR TITLE
Verup teleport 15.1.3 with golang 1.22.1 and node.js 18.19.1

### PR DESCRIPTION
**Pull request by takara9**
This PR is the part of the container regular update.
Rust , Wasm add in Dockerfile, due to teleport v15.x require them.

### CODE REVIEW

File: teleport-node/TAG [hunk 1]
```diff
@@ -1 +1 @@
-14.3.0.4
+14.3.7.1
```
Comment (yokaze) [inline teleport-node/TAG:1]: Could you please use the same upstream version between TAG and Dockerfile?
For the other changes, please consult umezawa-san.

```suggestion
15.1.3.1
```

Comment (umezawatakeshi) [inline teleport-node/TAG:1]: The tag is wrong.
Probably it should be:
```suggestion
15.1.3.1
```

File: teleport-node/Dockerfile [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+FROM ghcr.io/cybozu/golang:1.22-jammy AS build
+
+ARG TELEPORT_VERSION=15.1.3
+ARG RUST_VERSION=1.71.1
+ARG WASM_PACK_VERSION=0.12.1
 
-ARG TELEPORT_VERSION=14.3.0
 # Install Nodejs
-ARG NODE_VERSION=18.18.2
+ARG NODE_VERSION=18.19.1
```
Comment (umezawatakeshi) [inline teleport-node/Dockerfile:9]: According to the PR title, node.js version is 21.11.1. Which is correct?

**Review by umezawatakeshi — CHANGES_REQUESTED** [review]

File: teleport-node/Dockerfile (hunk 2 above)
Comment (takara9) [reply teleport-node/Dockerfile:9]: sorry, 18.19.1 is right. I revice the title

Comment (takara9) [reply teleport-node/Dockerfile]: I changed again to 18.19.2

**Comment by umezawatakeshi** [comment]
Do not leave commit message as is after you squashed some commits. The messages should describe what the commit does.

**Review by yokaze — APPROVED** [review]
LGTM if umezawa-san confirms of the other changes.

**Comment by takara9** [comment]
@umezawatakeshi 
Thank you for your advice, I update the latest commit message.

**Review by umezawatakeshi — CHANGES_REQUESTED** [review]
nothing resolved

**Comment by takara9** [comment]
@umezawatakeshi 
I updated the commit message like this "update teleport v15.1.3 with add rust and wasm in Dockerfile, and fix some typo by regular update"

I am sorry, please tell me your expectaoion about commit message.

**Comment by takara9** [comment]
@umezawatakeshi 
I update the commit message again.

**Comment by takara9** [comment]
I deleted historycal logs in the commit message.

**Review by umezawatakeshi — APPROVED** [review]

**Pull request merged**